pages/careers.tsx

```diff
@@ -59,8 +60,8 @@ export default function Careers() {
                 className={classNames(
                   contextIdx === 0 ? 'rounded-tl-lg rounded-tr-lg sm:rounded-tr-none' : '',
                   contextIdx === 1 ? 'sm:rounded-tr-lg' : '',
-                  contextIdx === context.length - 2 ? 'sm:rounded-bl-lg' : '',
-                  contextIdx === context.length - 1
+                  contextIdx === perks.length - 2 ? 'sm:rounded-bl-lg' : '',
```

Comment:
I'm assumming this is the check that was supposed to be there. Not completely sure tho.

Reply:
Did you check it local?